grixraz/reference.py

```diff
@@ -48,7 +53,13 @@ def create_references(
     ref_fn(files[0])
 
     with ThreadPoolExecutor(max_workers=n_workers) as executor:
-        results = list(tqdm(executor.map(ref_fn, files), desc = "Creating references...", total=len(files)))
+        results = list(
+            tqdm(
```

Comment:
Not related to this PR but just a note that the tqdm did not show any progress when I ran the demo notebook

Reply:
hmmm interesting, for me this works